### DISCUSSION TITLE
What exactly is a "Validation-Domain" warning?

### DISCUSSION THREAD

**Discussion by Trenly** [microsoft/winget-pkgs#33922] [Q&A] [answered]
Just wondering what this means, why it has to go to manual review, why it seems like there has been more of these lately, and how I can avoid this issue when submitting PR's. Basically just want to know more about what the warnings are/mean and the purpose of them

**Comment by Trenly** [comment]
@denelon I know MSFT has some holiday time coming up; Any chance that someone from pipelines could provide some additional insights here before their vacation?

**Comment by denelon** [comment, marked as answer]
There are a few domains we're prohibited from using for distribution like fosshub.com as it's clearly part of their terms of service. In other cases, we're looking for a match between the publisher and the domain the download is coming from. It's a fairly simple heuristic so we can try to deduce when something might be incorrect or inappropriate for distribution. There isn't a reputable authoritative source to make the determination in all cases, so we've built the waiver system so we can make sure someone has had a chance to look and provide a waiver.

**Reply by Trenly** [reply]
So basically, it looks to check that the installer URL comes from a domain which is associated with the publisher?
